validator/json/examples/hub-schedule2.json

```diff
@@ -213,7 +213,9 @@
         "title": "38C3: Opening Ceremony",
         "subtitle": null,
         "language": "en",
-        "track": "CCC",
+        "track": {
+          "slug": "ccc"
+        },
```

Comment:
I changed the track reference, to be uniform with the room refeference

Reply:
Are we missing a reference to the `guid` of the track here?

Reply:
I am not yet at the point where I see a track guid as really required, a persistent slug is a good enough "ID" for me for this kind of entity... Otherwise we have to discuss if they have the same guid across each conference instance etc.